tests/integration/crunchers/test_cruncher_quantile_rolling_windows.py

```diff
@@ -167,6 +167,24 @@ def test_numerical_relationship(self):
 
         assert all(crunched["value"].values == expected)
 
+    def test_extreme_values_relationship(self):
+        # Calculate the values using the cruncher for a fairly detailed dataset
+        large_db = IamDataFrame(self.large_db.copy())
+        tcruncher = self.tclass(large_db)
+        res = tcruncher.derive_relationship("Emissions|CH4", ["Emissions|CO2"])
+        assert callable(res)
+        crunched = res(large_db)
+
+        modify_extreme_db = large_db.filter(variable="Emissions|CO2").copy()
+        ind = modify_extreme_db["value"].idxmax
+        modify_extreme_db["value"].loc[ind] += 10
+        extreme_crunched = res(modify_extreme_db)
+        assert all(crunched["value"] == extreme_crunched["value"])
```

Comment:
can you put a comment above this explaining? Something like, "filling the df with the extreme value should give the same result as filling the df without the extreme value as our filler's extrapolation is constant"